bpf/process/bpf_execve_event.c

```diff
@@ -31,6 +31,17 @@ struct {
 	__type(value, struct msg_data);
 } data_heap SEC(".maps");
 
+/* tg_mbset_map holds a mapping from (binary) paths to a bitset of ids that it matches. The map is
+ * written by user-space and read in the exec hook to determine the bitset of ids of a binary that
+ * is executed.
+ */
+struct {
+	__uint(type, BPF_MAP_TYPE_HASH);
+	__uint(max_entries, 1024);
+	__type(key, __u8[MATCH_BINARIES_PATH_MAX_LENGTH]);
+	__type(value, mbset_t);
```

Comment:
it seems that you declare this map here with the type `mbset_t` but only define it a bit later in a7dbc8c7c5b8889baaaeb501c633eee9458448e2.

Reply:
Thanks, fixed!